Makefile

```diff
@@ -310,6 +312,16 @@ dist-tag: freeradius-server-$(RADIUSD_VERSION_STRING).tar.gz freeradius-server-$
 #
 #	Build a debian package
 #
+debian/control: debian/rules
+	debian/rules $?
+
+freeradius-build-deps_$(RADIUSD_VERSION_STRING)_all.deb: debian/control
+	mk-build-deps $?
+
+.PHONY: deb-build-deps
+deb-build-deps: freeradius-build-deps_$(RADIUSD_VERSION_STRING)_all.deb
+	apt-get install -y ./$?
```

Comment:
You can do this all in one step; simpler, and saves problems if the package name changes in future:

    mk-build-deps -ir debian/control

I'm slightly indifferent, but I'm not too sure we should have the Makefile installing (or trying to install) packages on the system? It's also encouraging someone to run `make` as root.

Reply:
@mattrose can you make the change @mcnewton suggested and then I'll merge the PR

Reply:
That change brings up the apt confirmation dialog.  Which means we then can't automate the procedure.

Reply:
@mattrose try:

`export DEBIAN_FRONTEND=noninteractive; mk-build-deps -ir debian/control`

or

`DEBIAN_FRONTEND=noninteractive mk-build-deps -ir debian/control`

Reply:
Ah! It is a makefile. the second option should work.

Reply:
TBH we should set `DEBIAN_FRONTEND` before all apt-get calls. But I'm surprised it stops for input, I've never seen it ask for confirmation before. Recent change in behaviour?

Reply:
Adding `export DEBIAN_FRONTEND=noninteractive` at the 314 should work fine.

Reply:
DEBIAN_FRONTEND doesn't get rid of the initial confirmation dialog, it just suppresses any interactive prompts from the packages themselves.

Reply:
The only way I can see to change that behaviour is to edit the apt config file itself, which I don't want to do

Reply:
Should be able to fix with the `-t` option then

Reply:
hmm, I'll give that a try